components/base-war/src/main/webapp/skins/phenotips/header.vm

```diff
@@ -41,6 +41,8 @@
     </a>
   </div> ## companylogo
   </div> ## company
+  #set ($displayLanguageChoice = true)
```

Comment:
`language.vm` has been copied&adapted from XWiki, so in order to have fewer changes to that file, we just define this variable here.